src/misc/help_util.cpp

```diff
@@ -0,0 +1,52 @@
+#include "dosbox.h"
+
+#include <map>
+#include <string>
+
+#include "help_util.h"
+#include "string_utils.h"
+#include "support.h"
+
+namespace HelpUtil {
```

Comment:
DOSBox historically hasn't used namespaces, because cross-file functions have been shared with:

`MODULE_SomeFunction`

For example:
`PCSPEAKER_SetCount`
`MIXER_...`
`BIOS_...`

Can we do the same here?

Reply:
I suppose so. Namespaces are such a nifty feature though!